README.md

```diff
@@ -44,13 +44,22 @@ If you'd like to try it out, navigate to [app.skyvern.com](https://app.skyvern.c
 
 ## Install & Run
 
+Dependencies needed:
```

Comment:
I think this clarification upfront makes a lot of sense for people looking into that for the first time.

On Win, def not working on Python 3.13 atm, so, also I think we should explicitly mention that to avoid confusion.